op-node/flags/p2p_flags.go

```diff
@@ -96,7 +96,7 @@ var (
 		Name:     "p2p.listen.ip",
 		Usage:    "IP to bind LibP2P and Discv5 to",
 		Required: false,
-		Value:    "0.0.0.0",
+		Value:    "0.0.0.0", // TODO: SHould this be 0.0.0.0? I think so b/c it is public
```

Comment:
Yes, p2p port should default to 0.0.0.0 because it's completely useless if it's not publicly available.